data/sql/updates/pending_db_world/rev_1557589006688722000.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1557589006688722000');
+
+INSERT INTO `spell_group` (`id`,`spell_id`) VALUES (1052,48384),(1058,20335),(1097,14892),(1098,16176);
```

Comment:
always use `DELETE` before `INSERT`, also please check why the CI pipeline is failing